apps/src/templates/manageStudents/ManageStudentsAgeCell.jsx

```diff
@@ -6,9 +6,9 @@ import {editStudent} from './manageStudentsRedux';
 class ManageStudentAgeCell extends Component {
   static propTypes = {
     id: PropTypes.number.isRequired,
-    age: PropTypes.number,
+    age: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     isEditing: PropTypes.bool,
-    editedValue: PropTypes.number,
+    editedValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Allowing both types here seems like a "code smell". Can we just convert this to a number in `onChangeAge`, and then expect it to be a number everywhere?

Reply:
right now it looks like the API in the current table sends both the number 4, and the string "21+" back to the server to update, hence the two different types.